akka-persistence-typed/src/main/resources/reference.conf

```diff
@@ -0,0 +1,11 @@
+akka.persistence.typed {
+
+  # default stash buffer size for incoming messages to persistent actors
+  stash-buffer-size = 1024
+
+  # enables automatic logging of messages stashed automatically by an PersistentBehavior,
+  # this may happen while it receives commands while it is recovering events or while it is persisting events
+  # Set to a log-level (debug, info, warn, error) to log stashed messages on given log level;
+  log-stashing = off
```

Comment:
why is the log level configurable? `log-stashing-enabled = on/off` ? or `verbose-logging = on/off`